core/arch/arm/plat-imx/drivers/tzc380.c

```diff
@@ -11,17 +11,35 @@
 #include <imx.h>
 #include <mm/core_memprot.h>
 #include <mm/generic_ram_layout.h>
+#include <config.h>
+
+/*
+ * Used during tzc380 initialization, should be removed by dead code
+ * eliminiation when set to 0.
```

Comment:
`should be removed by dead code elimination when set to 0.`
Rather `TZASC2_BASE is asserted non null when used.`